src/main.rs

```diff
@@ -23,15 +24,42 @@ const PLATFORM: &'static [PlatformData] = &[
 fn main() {
     let mut col = Collector::new();
 
+	let matches = App::new("conga")
+        .version("0.1.0")
+        .author("")
+        .about("System Configurator Gatherer")
+        .arg(Arg::with_name("verbose")
+            .short("v") // TODO: currently unused: set up logging first
+            .long("verbose")
+            .help("increase verbosity")
+            .takes_value(false))
```

Comment:
is there an option needed to be set for multiple instances like `-vvv`? if so, just set it now, don't wait for `--verbose` to actually be used

Reply:
added `.multiple(true)`, good enough for now. should definitely implement a centralized logger before we get too many `println!()`s everywhere